### PR TITLE
add README

### PR DESCRIPTION
**Pull request by xmulligan**
Adding the README to the repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+## Contribute
+
+Check out the [contributor ladder](CONTRIBUTOR-LADDER.md) for different ways to get involved and level up within the project. You can see different roles within the project in the [contributor roles](CONTRIBUTOR-ROLES.md). If you have already contributed to the project in any way, please add yourself to the [contributor file](CONTRIBUTORS.md).
+
+# Socializing
+
+Connect with the community through:
+
+* Follow us on [Twitter](https://twitter.com/ciliumproject) and our [blog](https://cilium.io/blog/)
+* Chat with us on [Slack](https://cilium.herokuapp.com/)
```
Comment (joestringer) [inline README.md:14]:
```suggestion
* Chat with us on [Slack](https://cilium.io/slack)
```

**Review by joestringer — COMMENTED** [review]
In case we ever end up changing the heroku app, I think the Slack link should point to cilium.io. Otherwise LGTM.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## Contribute
+
+Check out the [contributor ladder](CONTRIBUTOR-LADDER.md) for different ways to get involved and level up within the project. You can see different roles within the project in the [contributor roles](CONTRIBUTOR-ROLES.md). If you have already contributed to the project in any way, please add yourself to the [contributor file](CONTRIBUTORS.md).
+
+# Socializing
+
+Connect with the community through:
+
+* Follow us on [Twitter](https://twitter.com/ciliumproject) and our [blog](https://cilium.io/blog/)
```
Comment (qmonnet) [inline README.md:13]: Do we have a Mastodon alternative, as well?

**Review by qmonnet — APPROVED** [review]
Looks good, thanks!

One side question, is the Apache licence relevant for this repository? My impression was that it would cover code, and that something like Creative Commons would be better suited to text contents? (_I'm not a lawyer, etc._)

File: README.md (hunk 2 above)
Comment (xmulligan) [reply README.md:13]: No, but we do have LinkedIn so I'll add that

Comment (xmulligan) [inline README.md:13]:
```suggestion
* Follow us on [Twitter](https://twitter.com/ciliumproject), [LinkedIn](https://www.linkedin.com/company/cilium/), and our [blog](https://cilium.io/blog/)
```

**Comment by xmulligan** [comment]
Yes, I think technically most of our repos should be under Apache for code and CC for writing. For instance, the Cilium docs are under cilium/cilium which is just Apache. I used Apache for this since Kubernetes does the same for their [community repo](https://github.com/kubernetes/community). In the future, this repo will probably also contain community tooling.

**Review by aanm — APPROVED** [review]

**Pull request merged**